microsetta_interface/templates/new_results_page.jinja2

```diff
@@ -1245,7 +1284,23 @@
             }
         );
 
+        // First, hide all of the site-specific content
+        const all_site_specific_elements = document.getElementsByClassName("site_specific_content");
+        for (let i = 0; i < all_site_specific_elements.length; i++) {
+            all_site_specific_elements[i].style.display = "none";
+        }
+
+        // Then, display the site-specific content for our current dataset
+        const site_specific_elements = document.getElementsByClassName("site_specific_content-" + state.dataset_site.value);
+        for (let i = 0; i < site_specific_elements.length; i++) {
+            site_specific_elements[i].style.display = "";
+        }
 
+        // Hide Microbiome Maps for skin
+        if (state.dataset_site.value === "skin") {
+            document.getElementById("microbial-map-tab").parentElement.style.display = "none";
```

Comment:
Why can't the elements being handled in this if be assigned the site_specific_content class and hidden as part of the code directly above?

Reply:
Good catch, adjusted.